app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -1963,4 +1974,54 @@ private void searchFiles() {
   public boolean checkNull(View view) {
     return view != null;
   }
+  public static void sendNotification(String message,Context context){
+
+        Intent actionIntent_stop = new Intent(context, NotificationService.class)
+                .setAction(ACTION_STOP);
+        PendingIntent actionPendingIntent_stop = PendingIntent.getService(context, 0,
+                actionIntent_stop, PendingIntent.FLAG_ONE_SHOT);
+        Intent actionIntent_pause_play = new Intent(context, NotificationService.class)
+                .setAction(ACTION_PAUSE);
+        PendingIntent actionPendingIntent_pause_play = PendingIntent.getService(context, 0,
+                actionIntent_pause_play, PendingIntent.FLAG_ONE_SHOT);
+        NotificationCompat.Builder builder = new NotificationCompat.Builder(context)
+                .setSmallIcon(R.drawable.kiwix_icon_with_title)
+                .setContentTitle("Read Aloud")
+                .setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
+                .addAction(R.drawable.ic_stop_black_24dp, "stop",
+                        actionPendingIntent_stop)
+                .setPriority(Notification.PRIORITY_MAX);
+        if(message.equals("resume")) {
+            builder.addAction(R.drawable.ic_play_arrow_black_24dp, message,
+                    actionPendingIntent_pause_play);
+        }
+        else{
+            builder.addAction(R.drawable.ic_pause_black_24dp, message,
+                    actionPendingIntent_pause_play);
+        }
+        NotificationManager mNotificationManager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
```

Comment:
Do not use Hungarian notations. So it should be renamed to `notificationManager`.

Reply:
Okay Sure:) Let me check this.